unmaintained/repeated_vacuum/repeated_vacuum.go

```diff
@@ -35,7 +35,7 @@ func main() {
 	go func() {
 		for {
 			println("vacuum threshold", *garbageThreshold)
-			_, _, err := http_unknown.Get(fmt.Sprintf("http://%s/vol/vacuum?garbageThreshold=%f", pb.ServerAddress(*master).ToHttpAddress(), *garbageThreshold))
+			_, _, err := util_http.Get(util_http.GetGlobalHttpClient(), fmt.Sprintf("http://%s/vol/vacuum?garbageThreshold=%f", pb.ServerAddress(*master).ToHttpAddress(), *garbageThreshold))
```

Comment:
make this simpler?

```
util_http.Get(util_http.GetGlobalHttpClient(), "url")
```
into
```
util_http.Get("url")
```

Reply:
Chris, hello!

If I replace the function with a method, it will no longer:
- perform JWT authorization by token;
- check the content for gzip (I suspect that gzip is not needed in this case since no data is being transferred);
- handle StatusCode > 500.

The StatusCode check can be done where the function is called. If JWT authorization is not needed, I can change it to a Get method. Or did I misunderstand you?

Reply:
Hi @chrislusf,

I guess I misunderstood you. Did you mean to remove the client argument from the `util_http` functions?

Did I understand correctly that I need to remove the client argument for all functions in the `weed/util/http/http_util.go` file (return the arguments to their original state):
- `Post`;
- `Get`;
- `GetAuthenticated`;
- `Head`;
- `Delete`;
- `DeleteProxied`;
- `GetBufferStream`;
- `GetUrlStream`;
- `DownloadFile`;
- `Do`;
- `NormalizeUrl`;
- `ReadUrl`;
- `ReadUrlAsStream`;
- and so on....

Reply:
Hi @chrislusf,

Please let me know if there's anything else I need to do to have the PR accepted🙂